draft-ietf-quic-transport.md

```diff
@@ -5710,6 +5710,12 @@ that a peer is able to understand the frame.  An endpoint can use a transport
 parameter to signal its willingness to receive one or more extension frame types
 with the one transport parameter.
 
+Extensions that modify or replace core protocol functionality (including frame
```

Comment:
do we want to be specific about **this** version?

Reply:
We don't yet know about the portability of extensions across versions.  I'm kind of inclined to just leave that topic for future-us.

Reply:
+1, this document only discusses this version. Any other version is future work

Reply:
Makes sense, I agree.